<?xml version="1.0"?>
<launch>
    <arg name="arg_marker1_id" default="0" />
    <arg name="arg_marker2_id" default="1" />

    <node name="ps_test" pkg="phasespace" type="ps_test" output="screen">
        <param name="marker1_id" type="int" value="$(arg arg_marker1_id)" />
        <param name="marker2_id" type="int" value="$(arg arg_marker2_id)" />
    </node>
</launch>
